v2/api/dbformariadb/customizations/server_extensions.go

```diff
@@ -74,8 +74,8 @@ func secretsToWrite(obj *mariadb.Server) ([]*v1.Secret, error) {
 		return nil, errors.Errorf("unexpected nil OperatorSpec")
 	}
 
-	collector := secrets.NewSecretCollector(obj.Namespace)
-	collector.AddSecretValue(operatorSpecSecrets.FullyQualifiedDomainName, to.String(obj.Status.FullyQualifiedDomainName))
+	collector := secrets.NewCollector(obj.Namespace)
```

Comment:
I was confused for a moment, but now I get it - saying `secrets.NewCollector()` is definitely nicer than having to stutter when saying `secretes.NewSecreteCollector()`.

Reply:
Yeah I think this is a somewhat standard Go idiom, where you use the package name to help keep things short. @babbageclunk also did it and drew a double-take from me with `config.Values` (which is just `Values` when unqualified). This shortening drew inspiration from that.